### DISCUSSION TITLE
ModuleNotFoundError: No module named 'requests'

### DISCUSSION THREAD

**Discussion by SebastianSosa** [flet-dev/flet#3267] [Q&A]
Hi,

I created an app that runs correctly on Windows. However, when I create the APK and install it on my phone, I encounter the following error message:

Traceback (most recent call last):
  File "<string>", line 43, in <module>
  File "<frozen runpy>", line 229, in run_module
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\SSosa\AppData\Local\Temp\serious_python_temp70f602ae\main.py", line 3, in <module>
ModuleNotFoundError: No module named 'requests'

My requirements.txt contains requests.

I am on flutter 3.19.6, flet 0.22.0 and python 3.10.9.

Thank you.

**Comment by SebastianSosa** [comment]
With the minimum example I get the same issue (requirements.txt contains requests.):

    `import flet
    from flet import IconButton, Page, Row,    TextField, icons
    import requests

    def main(page: Page):
    page.title = "Flet counter example"
    page.vertical_alignment = "center"

    txt_number = TextField(value="0", text_align="right", width=100)

    def minus_click(e):
        txt_number.value = str(int(txt_number.value) - 1)
        page.update()

    def plus_click(e):
        txt_number.value = str(int(txt_number.value) + 1)
        page.update()

    page.add(
        Row(
            [
                IconButton(icons.REMOVE, on_click=minus_click),
                txt_number,
                IconButton(icons.ADD, on_click=plus_click),
            ],
            alignment="center",
        )
    )

    flet.app(target=main)`

**Comment by Pomurnik** [comment]
Solution from #3356 doesn't work. I have requests in pyproject.toml (tried also with requirements.txt) and still have a issue. But . After building apk requests libary in build/site-packages/ folder i see requests folder for every platform. But afeter apk instalation on andorid device there is no requests folder in files/flet/python_side_packages/.
